pkg/pillar/types/vaultmgrtypes.go

```diff
@@ -5,15 +5,14 @@ package types
 
 import (
 	"github.com/lf-edge/eve/api/go/info"
-	"time"
 )
 
 //VaultStatus represents running status of a Vault
 type VaultStatus struct {
-	Name      string
-	Status    info.DataSecAtRestStatus
-	Error     string
-	ErrorTime time.Time
+	Name   string
+	Status info.DataSecAtRestStatus
+	// ErrorAndTime provides SetErrorNow() and ClearError()
```

Comment:
But it is not the de-facto. Is is merely the most common one.
And it might be far from obvious that this stuct inclusion adds functions to the object.